stock_request/models/stock_request_order.py

```diff
@@ -320,6 +320,16 @@ def _check_location_company(self):
                 )
             )
 
+    @api.constrains("stock_request_ids")
+    def _check_location_empty_stock_request_ids(self):
+        if any(not request.stock_request_ids for request in self):
+            raise ValidationError(
+                _(
+                    """It is not possible to set empty stock request orders (maybe """
+                    """you should cancel it)."""
```

Comment:
No need of triple quotes:

```suggestion
                    "It is not possible to set empty stock request orders (maybe "
                    "you should cancel it)."
```